torch/_inductor/utils.py

```diff
@@ -1599,10 +1599,20 @@ def aoti_compile_with_persistent_cache(
     supported_scalar_types = tuple(type_to_torch_dtype.keys())
     flattened_inputs = pytree.arg_tree_leaves(*args, **kwargs)
     if not all(
-        isinstance(input, (supported_scalar_types, torch.Tensor))
+        isinstance(input, (supported_scalar_types, torch.Tensor, list))
         for input in flattened_inputs
     ):
-        raise NotImplementedError("Only support tensor, int, float, bool for now")
+        raise NotImplementedError(
+            "Only support tensor, tensor list, int, float, bool for now"
+        )
+
+    for input in flattened_inputs:
+        if isinstance(input, list) and not all(
```

Comment:
I'm a bit confused as to why `flattened_inputs` contains lists.  Shouldn't the call to `pytree` "flatten" all the lists away (hence the variable name)?

Reply:
Sorry for the late response. Refined the code by avoiding flattening all arguments.